package.json

```diff
@@ -274,6 +274,10 @@
 						"description": "List of additional resources.",
 						"uniqueItems": true
 					},
+					"secret": {
+						"type": "string",
+						"description": "Secret key. The secret is added as runtime configuration."
```

Comment:
I find this description a little confusing. The field is the "key" but the secret is added?